ui/lib/core/addon/components/read-more.js

```diff
@@ -6,7 +6,8 @@ import { tracked } from '@glimmer/tracking';
 
 /**
  * @module ReadMore
- * ReadMore components are used to wrap long text that we'd like to show as one line initially with the option to expand and read. Text which is shorter than the surrounding div will not truncate or show the See More button.
```

Comment:
This was hard to read as one line in the code, so pushed to two so you could see without scrolling horizontally.

Reply:
Thank you for cleaning up! 🧹

Reply:
If you decide to make that additional parent class, maybe we could mention it here?

Reply:
Good call. I'll add that here.